components/utilities/headerLink.module.css

```diff
@@ -10,6 +10,10 @@
   @apply font-bold;
 }
 
+.HeaderContainer code {
+  @apply text-red-70 rounded-md px-1 mx-1;
```

Comment:
I find the red text a bit too prominent in headings, especially with long commands:

![Screen Shot 2022-09-13 at 08 05 33](https://user-images.githubusercontent.com/34423371/189885733-00c87580-546f-43d3-a5a0-4608a1c7faac.png)

I'd keep it black

Reply:
On second pass, I agree. The red is too much in headings. I've reverted to black on b81567e